.projenrc.js

```diff
@@ -19,21 +19,21 @@ const project = new awscdk.AwsCdkConstructLibrary({
     distName: 'cdk-nag',
     module: 'cdk_nag',
   },
-  publishToNuget: {
-    packageId: 'Cdklabs.CdkNag',
-    dotNetNamespace: 'Cdklabs.CdkNag',
-  },
-  publishToMaven: {
-    mavenGroupId: 'io.github.cdklabs',
-    javaPackage: 'io.github.cdklabs.cdknag',
-    mavenArtifactId: 'cdknag',
-    mavenEndpoint: 'https://s01.oss.sonatype.org',
-  },
-  publishToGo: {
-    moduleName: 'github.com/cdklabs/cdk-nag-go',
-    gitUserName: 'cdklabs-automation',
-    gitUserEmail: 'cdklabs-automation@amazon.com',
-  },
+  // publishToNuget: {
+  //   packageId: 'Cdklabs.CdkNag',
+  //   dotNetNamespace: 'Cdklabs.CdkNag',
+  // },
+  // publishToMaven: {
+  //   mavenGroupId: 'io.github.cdklabs',
+  //   javaPackage: 'io.github.cdklabs.cdknag',
+  //   mavenArtifactId: 'cdknag',
+  //   mavenEndpoint: 'https://s01.oss.sonatype.org',
+  // },
+  // publishToGo: {
+  //   moduleName: 'github.com/cdklabs/cdk-nag-go',
+  //   gitUserName: 'cdklabs-automation',
+  //   gitUserEmail: 'cdklabs-automation@amazon.com',
+  // },
```

Comment:
This shouldn't be committed 😄 

```suggestion
  // publishToNuget: {
  //   packageId: 'Cdklabs.CdkNag',
  //   dotNetNamespace: 'Cdklabs.CdkNag',
  // },
  // publishToMaven: {
  //   mavenGroupId: 'io.github.cdklabs',
  //   javaPackage: 'io.github.cdklabs.cdknag',
  //   mavenArtifactId: 'cdknag',
  //   mavenEndpoint: 'https://s01.oss.sonatype.org',
  // },
  // publishToGo: {
  //   moduleName: 'github.com/cdklabs/cdk-nag-go',
  //   gitUserName: 'cdklabs-automation',
  //   gitUserEmail: 'cdklabs-automation@amazon.com',
  // },
```

Reply:
If the .projenrc.rc shoud'nt be committed, then projen should have set up the .gitignore.. However thats becomes a circular problem, becuase you need the .projen file..   I think what needs to happen is that this does'nt get modifyed.  I'd turned of the go/java builds becuase i dont' have them installed, but I will, so that i can do the builds..

Reply:
Generally changes to the `projen` configuration should be committed if you are intending to make changes to the CI/CD/ publishing process for `cdk-nag`.

In this case I assume the intent is not to have `cdk-nag` stop publishing to those languages (which would happen if these changes were merged in)

Reply:
done